app/bundles/ProjectBundle/Translations/en_US/messages.ini

```
@@ -19,3 +19,6 @@ mautic.project.popover.updated.on="Updated on %date%"
 mautic.project.popover.view.details="View details"
 mautic.project.popover.edit="Edit"
 mautic.project.popover.footer.label="Project"
+mautic.project.contentblock.heading="Turn campaign plans into action"
```

```suggestion
mautic.project.contentblock.heading="Turn your marketing plans into action"
```
I think we should try to not confuse the word Campaign (as in, the Mautic thing) with marketing campaigns. While there's some changes coming down the pipe in the future in this area, for now let's be clear what we're referring to.